strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/support/ArtifactEntrySearchCriteria.java

```diff
@@ -0,0 +1,63 @@
+package org.carlspring.strongbox.services.support;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class ArtifactEntrySearchCriteria
+{
+
+    private Integer uselessnessDays;
+
+    private Long minSizeInBytes;
+
+    public boolean isEmpty()
+    {
+        return uselessnessDays == null && minSizeInBytes == null;
+    }
+
+    public Integer getUselessnessDays()
+    {
+        return uselessnessDays;
+    }
+
+    public Long getMinSizeInBytes()
+    {
+        return minSizeInBytes;
+    }
+
+    public static final class Builder
+    {
+
+        private Integer uselessnessDays;
+        private Long minSizeInBytes;
```

Comment:
What is `minSizeInbytes` exactly?

Reply:
You're able to specify the minimun artifact size you want to cleanup. For example, everything bigger than 1048576 bytes (1MB). It's an optional property for the cron job.